test/api_integration/apis/index_patterns/runtime_fields_crud/delete_runtime_field/errors.ts

```diff
@@ -59,16 +59,6 @@ export default function ({ getService }: FtrProviderContext) {
           expect(response1.status).to.be(404);
         });
 
-        it('returns error when attempting to delete a field which is not a runtime field', async () => {
```

Comment:
Same as test above, I don't see a need for a separate error code when removing a non runtime field